src/Microsoft.Net.Http.Headers/CacheControlHeaderValue.cs

```diff
@@ -184,7 +184,7 @@ public override string ToString()
             if (_noCache)
             {
                 AppendValueWithSeparatorIfRequired(sb, NoCacheString);
-                if ((_noCacheHeaders != null) && (_noCacheHeaders.Count > 0))
+                if (_noCacheHeaders?.Count > 0)
```

Comment:
Doh, fair point. Mental lapse on these - reverting the CacheControlHeaders changes. There is a chance for better Roslyn behavior here with the syntax sugar to prevent the boxing in many cases such as this though...